test/src/api/expect-global/testElementText.js

```diff
@@ -0,0 +1,147 @@
+const assert = require('assert');
```

Comment:
can we add more expect chains over here like `match` which I think would be used more frequently?